src/tenant/tenant.module.ts

```
@@ -0,0 +1,27 @@
+import { Module, Global } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { TenantService } from './tenant.service';
+import { TenantController } from './tenant.controller';
+import { Tenant } from './entities/tenant.entity';
+import { TenantMiddleware } from './middleware/tenant.middleware';
```

ESLint

[error] 'TenantMiddleware' is defined but never used.

(@typescript-eslint/no-unused-vars)